### DISCUSSION TITLE
Allow anonymous users OS4

### DISCUSSION THREAD

**Discussion by YvesRoth** [OpenSlides/OpenSlides#6614] [Q&A]
There used to be a simple check box to allow users without an account to access for example the motions or the agenda. Does anyone know how you can do this in the newest version of OpenSlides?

Help is very much appreciated

**Comment by jsangmeister** [comment]
This is currently deactivated and planned to be removed completely in the future. In the context of OS4, there were multiple problems regarding this functionality and it was not used very much anyway. For more context see https://github.com/OpenSlides/openslides-client/issues/1381

**Comment by ostcar** [comment]
We discussed this and there will be an anonymous user. Probably similar to the anonymous user in OpenSlides 3.x. But it will require some changes to the client. I can not tell you, when it will be finished.

**Comment by jsangmeister** [comment]
Progress is tracked in the linked issue. Closing this.
